docs/changelog.txt

```diff
@@ -1361,6 +1361,7 @@ changelog.txt uses a syntax similar to RST, with a few special sequences:
 ## Lua
 - Added a new ``dfhack.console`` API
 - API can now wrap functions with 12 or 13 parameters
+- Added (chain) for [CHAIN_METAL_TEXT] armours ``library/lua/gui/materials.lua`` (affects the ``gui/create-item`` script)
```

Comment:
A few suggestions for changelog entries:

- You should add changelog entries to the "Future" version at the top of the file. This section was for version 0.44.02-alpha1.
- Lead with the script that will show the change, and put the script name in single backticks, not double, so a hyperlink to the script is generated by the doc parser.
- Follow the script/plugin name with a colon. This is how the doc parser groups related entries.
- Unless the change is completely internal, describe the change in behavior that the user will see. End users are the ones reading this changelog.

Example:
```
- `gui/create-item`: Added "(chain)" annotation text for armours with the [CHAIN_METAL_TEXT] flag set